package.json

```diff
@@ -19,5 +18,8 @@
     "jasmine-tests": "jasmine",
     "test": "npm run lint && npm run jasmine-tests && npm run validate",
     "validate": "node validation/markets-validator.js"
+  },
+  "dependencies": {
```

Comment:
As long as all the dayjs files are copied to the libs folder this is not a dependency. Af course you did the right thing, but as long #342 is still open we should only have `devDependencies`.